app/views/posts/_add_comment.html.erb

```diff
@@ -0,0 +1,9 @@
+<div>
+  <%= form_with url: "/users/#{@post.author_id}/posts/#{@post.id}/comments", scope: :comment do |format| %>
+    <div>
+        <%= format.label :Enter_Comment %><br>
+        <%= format.text_area :text, rows: 10, require: true %>
+        <%= format.submit "Comment" %>
+    </div>
+    <% end %>
+</div>
```

Comment:
- [**Optional**] With Rails, we have a unique default way of organizing our work hence, this form could have been created in the path **views/comments/new.html.erb**. This is really important, especially when scaling or debugging our work in the future.